internal/db/schema/migrations/postgres/14/01_event.up.sql

```diff
@@ -0,0 +1,217 @@
+begin;
+
+/*
+┌─────────────────┐                                  
+│iam_scope_global │                                  
+├─────────────────┤                                  
+│                 │                                  
+└─────────────────┘                                  
+         ┼                                           
+         │                                           
+         ┼                                           
+         ┼                                           
```

Comment:
This says that for every row in `iam_scope_global` there must be 1 and only 1 row in the `event_config` table. If that is the case, I don't see anything that enforces this. Currently, it allows 0 or 1.